src/sheet_metal_conus.py

```diff
@@ -77,6 +96,71 @@ def getColorString(self, longColor, verbose=False):
         if verbose: inkex.debug("  %s for color default value"%(hexColor))
         return hexColor
 
+    # Marker arrows
+    def makeMarkerstyle(self, name, rotate):
+        defs = self.xpathSingle('/svg:svg//svg:defs')
+        if defs == None:
+            defs = inkex.etree.SubElement(self.document.getroot(),inkex.addNS('defs','svg'))
+        marker = inkex.etree.SubElement(defs ,inkex.addNS('marker','svg'))
+        marker.set('id', name)
+        marker.set('orient', 'auto')
+        marker.set('refX', '0.0')
+        marker.set('refY', '0.0')
+        marker.set('style', 'overflow:visible')
+        marker.set(inkex.addNS('stockid','inkscape'), name)
+
+        arrow = inkex.etree.Element("path")
+        # messy but works; definition of arrows in beautiful DIN-shapes:
+        if name.startswith('ArrowDIN-'):
+            if rotate:
+                arrow.set('d', 'M 8,0 -8,2.11 -8,-2.11 z')
+            else:
+                arrow.set('d', 'M -8,0 8,-2.11 8,2.11 z')
+        if name.startswith('ArrowDINout-'):
+            if rotate:
+                arrow.set('d', 'M 0,0 16,2.11 16,0.5 26,0.5 26,-0.5 16,-0.5 16,-2.11 z')
+            else:
+                arrow.set('d', 'M 0,0 -16,2.11 -16,0.5 -26,0.5 -26,-0.5 -16,-0.5 -16,-2.11 z')
+        arrow.set('style', 'fill:#000000;stroke:none')
+        marker.append(arrow)
```

Comment:
This makes the arrow into a marker. stores it in inkscape defs section of svg file. But only need to do it if not already there. Style is DIN style arrows.